docs/src/main/asciidoc/opentelemetry-metrics.adoc

```diff
@@ -322,20 +336,25 @@ counter.add(1, // <2>
 
 <1> Create a `LongCounter` named `hello-metrics` with a description and unit.
 <2> Increment the counter by one.
-<3> Add an attribute to the counter. This will create a dimension called `attribute.name` with value `attribute value`.
+<3> Add an attribute to the counter.
+This will create a dimension called `attribute.name` with value `attribute value`.
 
-IMPORTANT: Each unique combination of metric name and dimension produces a unique time series. Using an unbounded set of dimensional data (many different values like a userId) can lead to a "cardinality explosion", an exponential increase in the creation of new time series. Avoid!
+IMPORTANT: Each unique combination of metric name and dimension produces a unique time series.
+Using an unbounded set of dimensional data (many different values like a userId) can lead to a "cardinality explosion", an exponential increase in the creation of new time series.
+Avoid!
 
 OpenTelemetry provides many other types of Counters: `LongUpDownCounter`, `DoubleCounter`, `DoubleUpDownCounter` and also Observable, async counters like `ObservableLongCounter`, `ObservableDoubleCounter`, `ObservableLongUpDownCounter` and `ObservableDoubleUpDownCounter`.
 
 For more details please refer to the https://opentelemetry.io/docs/languages/java/instrumentation/#using-counters[OpenTelemetry Java documentation about Counters].
 
 === Gauges
-Observable Gauges should be used to measure non-additive values. A value that can increase or decrease over time, like the speedometer on a car. Gauges can be useful when monitoring the statistics for a cache or collection.
+Observable Gauges should be used to measure non-additive values.A value that can increase or decrease over time, like the speedometer on a car.
```

Comment:
You dropped a space here before A value.

Reply:
Fixed.